src/API/comments.js

```diff
@@ -0,0 +1,27 @@
+import { client } from '../utils/fetchClient';
+
+export const getComments = async (postId) => {
+  const response = await client.get(`/comments?postId=${postId}`);
+
+  return response.data;
+};
+
+export const addCommentToServer = async (commentData) => {
+  const response = await client.post('/comments', commentData, {
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded'
+    }
+  });
+
+  return response.data;
+};
+
+export const deleteCommentFromServer = async (commentId) => {
+  const response = await client.delete(`/comments/${commentId}`, {
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded'
```

Comment:
Similar to the POST request, verify if the DELETE request requires the `Content-Type` header to be `application/x-www-form-urlencoded`. Typically, DELETE requests do not need a body, so this header might be unnecessary.